types/postcss-nested/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for postcss-nested 4.1
+// Project: https://github.com/postcss/postcss-nested#readme
+// Definitions by: Maxim Vorontsov <https://github.com/VorontsovMaxim>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
```

Comment:
Which 2.8 features are used in this definition?  If there aren't any, it would be nice to specify a lower version number so that more projects can consume these types.